test/connectors/EventReportingConnectorSpec.scala

```diff
@@ -63,7 +63,7 @@ class EventReportingConnectorSpec
           )
       )
 
-      connector.getEventReportSummary(pstr) map { response =>
+      connector.getEventReportSummary(pstr, "21/01/22") map { response =>
```

Comment:
Comment r.e. this date again, won't repeat, just want to understand significance